platform/viewer/src/routes/WorkList/WorkList.jsx

```diff
@@ -234,7 +234,7 @@ function WorkList({ history, data: studies, isLoadingData, dataSource }) {
                 {date &&
                   moment(date, ['YYYYMMDD', 'YYYY.MM.DD'], true).isValid() &&
                   moment(date, ['YYYYMMDD', 'YYYY.MM.DD']).format(
-                    'MMM-DD-YYYY'
+                    'DD-MMM-YYYY'
```

Comment:
Incorrect change for ticket: https://precisionmetrics.atlassian.net/secure/RapidBoard.jspa?rapidView=1&projectKey=OHIF&modal=detail&selectedIssue=OHIF-167